src/integrations/Posthog/browser.js

```diff
@@ -46,74 +46,74 @@ class Posthog {
   }
 
   init() {
-    !(function (t, e) {
-      let o;
-      let n;
-      let p;
-      let r;
-      e.__SV ||
-        ((window.posthog = e),
-        (e._i = []),
-        (e.init = function (i, s, a) {
-          function g(t, e) {
-            const o = e.split('.');
-            o.length == 2 && ((t = t[o[0]]), (e = o[1])),
-              (t[e] = function () {
-                t.push([e].concat(Array.prototype.slice.call(arguments, 0)));
-              });
-          }
-          ((p = t.createElement('script')).type = 'text/javascript'),
-            (p.async = !0),
-            p.setAttribute('data-loader', LOAD_ORIGIN),
-            (p.src = `${s.api_host}/static/array.js`),
-            (r = t.getElementsByTagName('script')[0]).parentNode.insertBefore(p, r);
-          let u = e;
-          for (
-            void 0 !== a ? (u = e[a] = []) : (a = 'posthog'),
-              u.people = u.people || [],
-              u.toString = function (t) {
-                let e = 'posthog';
-                return a !== 'posthog' && (e += `.${a}`), t || (e += ' (stub)'), e;
-              },
-              u.people.toString = function () {
-                return `${u.toString(1)}.people (stub)`;
-              },
-              o =
-                'capture identify alias people.set people.set_once set_config register register_once unregister opt_out_capturing has_opted_out_capturing opt_in_capturing reset isFeatureEnabled onFeatureFlags'.split(
-                  ' ',
-                ),
-              n = 0;
-            n < o.length;
-            n++
-          )
-            g(u, o[n]);
-          e._i.push([i, s, a]);
-        }),
-        (e.__SV = 1));
-    })(document, window.posthog || []);
-
     const { POSTHOG } = this.analytics.loadOnlyIntegrations;
+    if (POSTHOG && !POSTHOG.loadIntegration) {
+      logger.debug('===[POSTHOG]: loadIntegration flag is disabled===');
+    } else {
```

Comment:
Instead of `if then else` do `if then return`

Reply:
updated